### DISCUSSION TITLE
Game Info screen

### DISCUSSION THREAD

**Discussion by renieness** [SwipesForScience/SwipesForScience#121] [Show and tell]
Could I please get some feedback on this screen? 

![Game Info](https://user-images.githubusercontent.com/39166059/111582690-4f39b100-880f-11eb-96f7-f26f308a003e.png)

I took the initial design for the game info and applied it to @jyeatman's Rapid Online Assessment of Reading game. 

I added a "Created by" section to give some authenticity to the game, and link back to the lab that spun it up.
I'm also wondering whether to show pictures of the researchers behind the game in that grey section, or not. Would it differ between education games & community games?

Please feel free to give feedback on other things as well e.g. the footer, text, etc.

**Comment by jyeatman** [comment]
I think that looks beautiful! In terms of the footer text, I think we could remove the "Swipes for Science is an open source..." and just fill that area with the logo or something

**Comment by erwingame** [comment]
Nice idea
